From f926559d4e801cebee0321b4ca4290d3c6223324 Mon Sep 17 00:00:00 2001
From: Lili Deng <lildeng@microsoft.com>
Date: Mon, 1 Dec 2025 15:08:25 +0800
Subject: [PATCH] xdpdump: fix complie issue

---
 lisa/microsoft/testsuites/xdp/xdpdump.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/lisa/microsoft/testsuites/xdp/xdpdump.py b/lisa/microsoft/testsuites/xdp/xdpdump.py
index 58f1a1c4a..9ec3dae65 100644
--- a/lisa/microsoft/testsuites/xdp/xdpdump.py
+++ b/lisa/microsoft/testsuites/xdp/xdpdump.py
@@ -208,6 +208,13 @@ class XdpDump(Tool):
                 cflags = f"{cflags} -D __PERF__"
             env_variables["CFLAGS"] = cflags
 
+        # Fix Makefile compilation rule
+        self.node.tools[Sed].substitute(
+            "-O2 -emit-llvm -c -g \\$<",
+            "-O2 -emit-llvm -g $< -o ${<:.c=.ll}",
+            f"{self._code_path}/Makefile",
+            sudo=True,
+        )
         make = self.node.tools[Make]
         make.make(
             arguments="",
-- 
2.51.0

