From 09e0f176318dece8d73b46345d61c6085e77bf59 Mon Sep 17 00:00:00 2001
From: Rachel Menge <rachelmenge@microsoft.com>
Date: Tue, 2 Dec 2025 17:15:03 -0800
Subject: [PATCH] Add netperf support for Fedora

---
 lisa/tools/netperf.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lisa/tools/netperf.py b/lisa/tools/netperf.py
index 29ca24ca0..88cdf7690 100644
--- a/lisa/tools/netperf.py
+++ b/lisa/tools/netperf.py
@@ -4,7 +4,7 @@
 from typing import Any, List, Type, cast
 
 from lisa.executable import Tool
-from lisa.operating_system import BSD, CBLMariner, Debian, Posix, Redhat, Suse
+from lisa.operating_system import BSD, CBLMariner, Debian, Fedora, Posix, Redhat, Suse
 from lisa.util import LisaException
 from lisa.util.process import Process
 
@@ -76,6 +76,8 @@ class Netperf(Tool):
         if not self._check_exists():
             if isinstance(self.node.os, BSD):
                 self.node.os.install_packages("netperf")
+            elif isinstance(self.node.os, Fedora):
+                self.node.os.install_packages("netperf")
             else:
                 self._install_from_src()
         return self._check_exists()
-- 
2.51.0

